main/src/main/java/org/mobilitydata/gtfsvalidator/validator/TimepointTimeValidator.java

```diff
@@ -32,13 +32,17 @@
  * <ul>
  *   <li>{@link StopTimeTimepointWithoutTimesNotice} - a timepoint does not specifies arrival_time
  *       or departure_time
+ *   <li>{@link MissingTimepointValue} - value for {@code stop_times.timepoint} is missing
  * </ul>
  */
 @GtfsValidator
 public class TimepointTimeValidator extends SingleEntityValidator<GtfsStopTime> {
 
   @Override
   public void validate(GtfsStopTime stopTime, NoticeContainer noticeContainer) {
+    if (!stopTime.hasTimepoint()) {
```

Comment:
Does the validator fill the column with a default value if it is not in the dataset, or are we not entering this `TimepointTimeValidator`?

Reply:
> Does the validator fill the column with a default value if it is not in the dataset,

Yes, default is `1`.